Bio/PDB/MMCIF2Dict.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, filename):
             i = 0
             n = 0
             for token in tokens:
-                if token == "loop_":
+                if token.lower() == "loop_":
```

Comment:
Should we raise an error (or at least a warning) if a non-lowercase "loop_" is found?

Reply:
While a little unusual, the formal spec allows any use of upper/lower case in the word so I don't think it should give a warning or error.

https://www.iucr.org/resources/cif/spec/version1.1/cifsyntax

Reply:
Agreed, https://www.iucr.org/resources/cif/spec/version1.1/cifsyntax is very clear that any case is allowed - even ``lOoP_`` would be valid.